### DISCUSSION TITLE
Hyperlinks wrong colours

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#532] [General] [answered]
![image](https://user-images.githubusercontent.com/64191883/154964860-084ca172-11a2-4af8-bd8f-7ad43fc18f59.png)

The hyperlinks colours does not match up to the house style. 

Normal : 
![image](https://user-images.githubusercontent.com/64191883/154964981-29dc7f49-c01e-4538-819b-94ccf8a0d8d0.png)

Clicked [Option, we've not determined a secondary color according to guidelines page and the green is too close to normal so i strongly suggest this blue [Information blue from guidelines and MUI(#1976D2)] for time of being]: 
![image](https://user-images.githubusercontent.com/64191883/154965763-855e779c-37ab-45ba-8149-26b9d4df19b4.png)

**Comment by techsmyth** [comment, marked as answer]
Hi Rosa, I have moved this to a card on the backlog: https://github.com/alkem-io/client-web/issues/1957

Also suggest that you pick it up, ok?! Cheers, Neil

**Reply by techsmyth** [reply]
Think this can be closed given you are solving it yourself :)
